### PR TITLE
Move bookmark folders plural to stringsdict

### PR DESCRIPTION
**Pull request by THISISDINOSAUR**
Task/Issue URL:
Tech Design URL:
CC:

**Description**:
Just some clean up

**Steps to test this PR**:
1. Test deleting a bookmarks folder with 1 item and the text is correct
2. Test deleting a bookmarks folder with 2 or more items and the text is correct
3. Repeat the above in other languages
4. Test other plurals haven't been messed up (e.g. trackers blocked dax dialog in onboarding (e.g. go through initial onboarding and then to cnn.com to get the dax dialog with multiple trackers))

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPad

**OS Testing**:

* [ ] iOS 13
* [ ] iOS 14
* [ ] iOS 15

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**

### CODE REVIEW

**Review by amddg44 — COMMENTED** [review]
In terms of the bookmark folder deletion messages, I tested in a few different languages and looks good 👍 

A few small issues have crept into some of the translations of `dax.onboarding.browsing.multiple.trackers` though!

- DE: all line breaks are gone
- CS: one of the line breaks has been replaced by html linebreak tags
- SK: line breaks replaced by html linebreak tags and most of the message is now in bold

image image image

**Comment by THISISDINOSAUR** [comment]
Thanks a lot for reviewing! I'd fixed this in one language, but didn't realise it'd happen so broadly. I'll have to fix it on Smartling I think

**Comment by THISISDINOSAUR** [comment]
CS uses a mix of correct and incorrect line breaks 😂 
Screenshot 2022-02-24 at 14 35 17

**Comment by THISISDINOSAUR** [comment]
This should be fixed now

File: DuckDuckGo/sk.lproj/Localizable.stringsdict [hunk 1]
```diff
@@ -13,29 +33,29 @@
...
-			<string>*%2$@, %3$@* a *1 ďalší* sa vás tu pokúsil sledovať.
+			<string>*%2$@, %3$@* a *1 ďalší* sa vás tu pokúsili sledovať. 
 
 Boli zablokovaní!
 
-☝️ Môžete skontrolovať panel s adresou URL a zistiť, kto sa vás snaží sledovať, keď navštívite novú webovú stránku.️</string>
+ ☝️Môžete skontrolovať panel s adresou a zistiť, kto sa vás snaží sledovať, keď navštívite novú webovú lokalitu.️</string>
 			<key>few</key>
-			<string>*%2$@, %3$@* a%1$d ďalší* sa vás tu pokúsili sledovať. 
+			<string>*%2$@, %3$@* a *%1$d ďalší sa vás tu pokúsili sledovať.
```
Comment (amddg44) [inline DuckDuckGo/sk.lproj/Localizable.stringsdict]: The closing `*` has been lost here so most of the message is bolded

image

File: DuckDuckGo/sk.lproj/Localizable.stringsdict [hunk 2]
```diff
@@ -13,29 +33,29 @@
...
-			<string>*%2$@, %3$@* a%1$d ďalší* sa vás tu pokúsili sledovať. 
+			<string>*%2$@, %3$@* a *%1$d ďalší sa vás tu pokúsili sledovať. 
 
 Boli zablokovaní!
 
- ☝️ Môžete skontrolovať panel s adresou URL a zistiť, kto sa vás snaží sledovať, keď navštívite novú webovú stránku.️</string>
+ ☝️Môžete skontrolovať panel s adresou a zistiť, kto sa vás snaží sledovať, keď navštívite novú webovú lokalitu.️</string>
 			<key>many</key>
-			<string>*%2$@, %3$@* a *%1$d ďalších* sa vás tu pokúsila sledovať. 
+			<string>*%2$@, %3$@* a *%1$d ďalší sa vás tu pokúsili sledovať.
```
Comment (amddg44) [inline DuckDuckGo/sk.lproj/Localizable.stringsdict]: Missing `*` here too

File: DuckDuckGo/sk.lproj/Localizable.stringsdict [hunk 3]
```diff
@@ -13,29 +33,29 @@
...
-			<string>*%2$@, %3$@* a *%1$d ďalších* sa vás tu pokúsila sledovať. 
+			<string>*%2$@, %3$@* a *%1$d ďalší sa vás tu pokúsili sledovať. 
 
 Boli zablokovaní!
 
- ☝️ Môžete skontrolovať panel s adresou URL a zistiť, kto sa vás snaží sledovať, keď navštívite novú webovú stránku.️</string>
+ ☝️Môžete skontrolovať panel s adresou a zistiť, kto sa vás snaží sledovať, keď navštívite novú webovú lokalitu.️</string>
 			<key>other</key>
-			<string>*%2$@, %3$@* a *%1$d ďalších* sa vás tu pokúsilo sledovať. 
+			<string>*%2$@, %3$@* a *%1$d ďalší sa vás tu pokúsili sledovať.
```
Comment (amddg44) [inline DuckDuckGo/sk.lproj/Localizable.stringsdict]: And final missing one!

**Review by amddg44 — CHANGES_REQUESTED** [review]
Looking good! Just a tweak to SK needed to correctly attribute text 👍

**Comment by THISISDINOSAUR** [comment]
Sorry about that! I missed the bold when I was checking it. We might need to do something to stop this happening again in the future, since we can't check every string for every language when it's xliffs. I'm not sure what though

**Review by amddg44 — APPROVED** [review]
No worries, it was easy to miss! And agree that we will need to start discussing a better long term strategy here at some point soon

**Pull request merged**